### PR TITLE
Add a Responsive layouts in Figma topic

### PR DESCRIPTION
**Pull request by MayaKirova**

### CODE REVIEW

File: en/best-layout-practices.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    It is specific to each tool and you can find more information in their related topics:
+
+  - [Responsive layouts in Figma](./best-layout-practices-figma.md)
+  - [Responsive layouts in Sketch](./best-layout-practices-sketch.md)
+  - [Responsive layouts in XD](./best-layout-practices-xd.md)
+  
+  As with this approach there's more available information from the design on how things should react to resize it would always give generation results closer to the original design in the App builder. 
+  Due to this it's recommended to utilize the specific tool's responsive layouting option to get better results.
+
+# Absolute Layout - Best Practices
```
Comment (dkamburov) [inline en/best-layout-practices.md:36]: Shouldn't the header be **Best Practices** only, doesn't seem to be related to **Absolute Layout**

File: en/best-layout-practices.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+This section provides design guidance regarding the positioning of Components in your app layouts. By sticking to the recommendations below, you will be able to generate an app effortlessly with the responsive behavior.
```
Comment (dkamburov) [inline en/best-layout-practices.md:38]: Maybe remove `This section provides guidance`, there is the same phrase two sentences bellow

File: en/best-layout-practices.md (hunk 1 above)
Comment (MayaKirova) [reply en/best-layout-practices.md:36]: Because it concerns how to group things in and Absolute layout design scenario.

Comment (MayaKirova) [reply en/best-layout-practices.md:36]: Best Practices for Auto-layouts are in the specific tool's topics - Responsive layouts in Figma/Sketch/Xd

File: en/best-layout-practices-figma.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+## Layout
+
+In Figma you can either use absolute positioned frames/groups, in which case layout generation will follow the [Absolute Layout](./best-layout-practices.md#layout) rules, or you can take advantage of the Auto-Layout options:
+
+### Auto-Layout
+
+Auto-layout is an unique Figma feature that allows structuring containers (such as Frames, Components, Groups) so that they have a more responsive layout that adapts according to the screen size. It allows setting rules on how the content should flow, how it should be positioned, sized and how it should spread within its container. As such it is very similar in concept as flexbox  in web.
+
+Auto-layout option can be set by the designed via Figma's UI:
```
Comment (dkamburov) [inline en/best-layout-practices-figma.md:20]:
```suggestion
Auto-layout option can be set by the designer via Figma's UI:
```

**Review by dkamburov — APPROVED** [review]
